requirements.txt

```diff
@@ -1,4 +1,4 @@
 mock
 nose
 pytest-django
-statsd==3.2.1
+statsd>=3.2.2.6
```

Comment:
let's drop this requirement back to `statsd>=3.3.0`, then pinning in our other repos with our pystatsd should satisfy, instead of pinning the version here and in our other repos